src/slo/sli_handlers.py

```diff
@@ -14,13 +14,19 @@ def get(self):
 
     @staticmethod
     def create_slo_recalculation_tasks():
-        return [
+        tasks = [
+          Tasks.create(method='POST',
+                       url='/sli/latency_for_x_days',
+                       params={'x_days': x_days})
+          for x_days in [0, 3, 4, 5, 7]
+        ]
+        tasks.append(
             Tasks.create(
                 method='POST',
-                url='/sli/latency_for_x_days',
-                params={'x_days': x_days})
-            for x_days in [0, 3, 4, 5, 7]
-        ]
+                url='/sli/quality',
+                params={}))
```

Comment:
I believe, you don't need to provide params if its empty

Reply:
done